### ISSUE TITLE
Tab box sliding

### ISSUE THREAD

**Issue by iGyver** [bwinton/TabCenter#899]
The behavior is just strange. The Browser box slides behind the tab and than after the move of the mouse above the rest of the browser box. 
So the tab slider changes (optical) the position behind the browser box. 

Please see the vid: [Link](https://vimeo.com/200131911)

**Comment by SoftVision-PaulOiegas** [comment]
Ok... What would be your expected behavior here ? It seams pretty logic to me since you still have the focus on the sidebar when you choose to hide it. 
Do you suggest that when you choose to hide it, the sidebar should be automatically hidden ?

**Comment by iGyver** [comment]
When I push the button I expected to shrink the Tab Center immediately. I pushed the button but the Browser box moved behind the Tab box. 

It need me a moment to realize that I have to move the mouse to the browser box for the style I was expecting. 

-- 
Ciao
Kay

**Comment by SoftVision-PaulOiegas** [comment]
I understand, and yeah, it could be an improvement. But as I said, mouse is still focusing the sidebar after you click the button. So somehow it's logic to keep it expanded. 
We'll see what the developers think of this. Thanks for the idea !

**Comment by iGyver** [comment]
Ok, I just don't think it's logical because I decided already to shrink it. :)

-- 
Ciao
Kay

**Comment by fvsch** [comment]
Agreed that if the user decides to shrink the tabs sidebar, it should bypass the "show expanded when hovered" behavior, to provide better visual feedback to the action.

And for users who were not sure what this button would do (like I was for a long time ^^), it would immediately demonstrate the effect. Helps learning the UI, in my opinion.
